### PR TITLE
style: modify styles on personal home page

### PR DESCRIPTION
**Pull request by sherotree**
## What type of PR is this?

- [x] Style
- [ ] Feature
- [ ] Bugfix
- [ ] Test
- [ ] Documentation
- [ ] Refactor
- [ ] Chore

## What this PR does / why we need it:
1. Remove '...' from more
**before:**
image
**current:**
image

2. Add [ #070A1A 60% ] as color-text-light-desc to files that named _color.scss and utils.scss
**before:**
image
**current:**
image

3. Align card contents
**before:**
image
**current:**
image

4. update '>>' to 'double-right-icon', update Icon to ErdaIcon
**before:**
image
**current:**
image

## Which issue(s) this PR fixes:

Fixes #

## Does this PR introduce a user interface change?

- [ ] Yes(screenshot is required)
- [ ] No

### CODE REVIEW

File: shell/app/config-page/components/text/text.tsx [hunk 1]
```diff
@@ -84,12 +85,12 @@ const Text = (props: CP_TEXT.Props) => {
                     className={`${textClassNames} hover-active`}
                   >
                     {tText}
-                    {icon && <CustomIcon className={`mr4 ml4 ${iconStyleName}`} type={icon} />}
-                    {image && <img src={image.startsWith('/images') ? imgMap[image] : image} className='text-image' />}
+                    {icon && <ErdaIcon type="Icon" props={{ iconType: icon, className: `mr4 ml4 ${iconStyleName}` }} />}
```
Comment (McDaddy) [inline shell/app/config-page/components/text/text.tsx:88]: the usage Of `ErdaIcon` is so strange... 
@michael8512  seems `type` is not using, why do we need pass a nested props？

Comment (michael8512) [reply shell/app/config-page/components/text/text.tsx:88]: It is a component of config-page

Comment (McDaddy) [reply shell/app/config-page/components/text/text.tsx:88]: > It is a component of config-page

so what? I didn't find out usage of `type="Icon"`

image

Comment (michael8512) [reply shell/app/config-page/components/text/text.tsx:88]: type can be removed

Comment (McDaddy) [reply shell/app/config-page/components/text/text.tsx:88]: > type can be removed

My key point is why do we need to pass a nested props. what's the difference if flatten the props？

File: shell/app/config-page/components/text/text.tsx [hunk 2]
```diff
@@ -84,12 +84,14 @@ const Text = (props: CP_TEXT.Props) => {
                     className={`${textClassNames} hover-active`}
                   >
                     {tText}
-                    {icon && <CustomIcon className={`mr4 ml4 ${iconStyleName}`} type={icon} />}
+                    {icon && <ErdaIcon iconType={icon
+                    } className={`mr4 ml4 ${iconStyleName}`} />}
```
Comment (daskyrk) [inline shell/app/config-page/components/text/text.tsx:88]: should be one line

**Review by daskyrk — APPROVED** [review]

**Pull request merged**